code/game/gamemodes/gang/dominator.dm

```diff
@@ -53,7 +53,7 @@
 			if(!warned && (time_remaining < 180))
 				warned = 1
 				var/area/domloc = get_area(loc)
-				gang.message_gangtools("Less than 3 minutes remain in hostile takeover. Defend your dominator at [domloc.map_name]!")
+				gang.message_gangtools("Less than 3 minutes remains in hostile takeover. Defend your dominator at [domloc.map_name]!")
```

Comment:
Inserting an auxiliary verb is not relevant to the change I made because the auxiliary verb, rather than the main verb, is required to agree with the subject when it is present.

`*Less (time) than 3 minutes do remain in hostile takeover.`
`Less (time) than 3 minutes does remain in hostile takeover.`

[minutes] being plural is not relevant because it is part of the modifier of the subject that the main verb must agree with. Consider this example with an explicit argument of [less] and a singular noun in the prepositional phrase in the subject:
`Less water than 1 liter remains in the bucket.`
`*Less water than 1 liter remain in the bucket.`

The prepositional phrase can also be made implicit, and the properties of the noun phrase it modifies will remain in compliance with the Projection Principle:
`Less time remains in hostile takeover.`
`Less water remains in the bucket.`
`*Less time remain in hostile takeover.`
`*Less water remain in the bucket.`